playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -76,6 +76,20 @@
   - "restart edxapp"
   - "restart edxapp_workers"
 
+- name: checkout microsites
+  git: >
+    dest="{{ EDXAPP_MICROSITE_ROOT_DIR }}/{{ item.key }}"
+    repo="{{ item.value.deploy.repo }}"
```

Comment:
This assumes that a new key, `deploy`, is added to a microsite's existing configuration. The key should take the form:

```python
microsite_configuration = {
    'OLI': {
        '...',
        'deploy': {
            'repo': 'https://github.com/ORG/REPO.git',
            'version': 'origin/rc',
        },
    },
}
```